src/main/java/io/cdap/plugin/python/transform/Py4jPythonExecutor.java

```diff
@@ -106,8 +108,8 @@ private KeyStore generatePemFileAndKeyStore(String transformTempDirString) throw
 
   private File prepareTempFiles() throws IOException, UnrecoverableKeyException,
     CertificateEncodingException, NoSuchAlgorithmException, KeyStoreException {
-    URL url = getClass().getResource("/pythonEvaluator.py");
-    String scriptText = new String(Files.readAllBytes(Paths.get(url.getPath())), StandardCharsets.UTF_8);
+    InputStream url = getClass().getResourceAsStream("/pythonEvaluator.py");
+    String scriptText = IOUtils.toString(url, StandardCharsets.UTF_8);
```

Comment:
does this close the InputStream too?

Reply:
Good catch! this doesn't close the stream. made changes to close the stream.